content/Api/Blip.Api.Template.Tests/Strategies/ExceptionHandlingStrategies/ApiExceptionHandlingStrategyTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Threading.Tasks;
+
+using Blip.Api.Template.Facades.Strategies.ExceptionHandlingStrategies;
+using Blip.Api.Template.Tests.Setup.Controller;
+
+using FluentAssertions;
```

Comment:
the entire project is using `Shouldly` to test. Why change it now?

Reply:
Just a preference. I will return to `Shouldly `to keep the same package already used.